src/App.tsx

```diff
@@ -2,14 +2,17 @@ import React from 'react';
 import {BrowserRouter as Router, Routes, Route} from 'react-router-dom';
 import './App.scss';
 import Home from './Home/Home';
+import Header from './header';
 import { Quiz } from './Quiz/Quiz';
 import Footer from './Footer/Footer';
 
+
 function App() {
   return (
     <div className="App">
       <Router>
         <h1>MarsioKart</h1>
+        <div><Header/></div>
```

Comment:
If you remove the `<div>`s from this line and just have `<Header/>` it will be easier for a screen reader to see what is happening and translate it - please can you do that?